blockstore/apps/core/migrations/0003_alter_user_first_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.7 on 2021-10-05 12:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Added by upgrade to Django 3.2. The only change seems to be the change in `max_length`.